PRINCIPLES.md

```diff
@@ -1,259 +1,62 @@
-The utility of having the entire system defined in a single canonical location grows in proportion to the complexity and internal coupling of the system.
-A web of references to configuration data located in different locations is undesirable, as it makes understanding the desired state particularly difficult.
-
-Versions should be uniquely named. This need not be a semantically meaningful name. It is sufficient that each new version is attributed a name that identifies it uniquely. Once a new version has been created, it should be immutable. By this we mean that it should be impossible to modify the relationship between a version's unique name and its value of the Desired State. 
-
-All but the very first version should reference a predecessor or parent, which is another uniquely named version. This enables us to retain a history of the changes.
-
-<!--
-#### Why is it necessary to have versions be immutable and retained indefinitely?
-
-
-
-- only verb is to create a new version.
-This - Only uniquely named new versions of the system configuration can be created and all configuration info is retained (forward only)
-- rollback
-- Audit
-The only exception being in the case of sensitive information leaks due to error.
-  (I would agrue that key rotation is always better option in this case, but for PII and proprietary info this isn't possible).
-- changes as transactions
-
-#### What are the responsibilities of a State store?
-- In addition, state stores must preserve metadata information regarding the identity of agents that create new versions.
-The Desired State of the new version, Who created it, when it was created and ideally, also _why_ it was created. - What Why, who when. 
-
-- Even though we say git, it's actually only true when configured in a very particular way.
-
-#### State stores as coordination points/ collab via common data interface
-- Collab point. 
-- Business rules and security rules about the acceptable state of the system shoudl be checked against any new proposed version of the desired configuration before such changes are accepted as a new version. (Gates before version)
-- where human process is integrated
-- the place where collaboration can be synchronised.
--->
-
-### 3. Continuous state reconciliation
-
-<div style="background: #E3F2FD; border-left: 3px solid #0D47A1; padding:10px; margin-bottom:5px; font-style:italic">
-Software agents continuously, and automatically, compare a system's <em>Actual State</em> to its <em>Desired State</em>.
-If the actual and desired states differ, automated actions are immediately attempted to reconcile them.
-These differences could be due to the actual state drifting from the desired state, or the desired state changing intentionally.
-</div>
+    If the actual and desired states differ for any reason, automated actions to reconcile them are initiated.
```

Comment:
```suggestion
    If the actual and desired states differ for any reason, automated actions to reconcile them are initiated.
     
    By "continuous" we adopt the industry-standard term to mean reconciliation continues to happen, not that it must be instantaneous
```

Reply:
This is being moved to the Glossary section